lms/templates/financial-assistance/financial_assessment_form.underscore

```diff
@@ -39,7 +39,7 @@
 	<%= HtmlUtils.HTML(fieldsHtml) %>
 
 	<div class="cta-wrapper clearfix">
-		<a href="<%- student_faq_url %>" class="nav-link"><%- edx.HtmlUtils.interpolateHtml(
+		<a href="<%- student_faq_url %>" class="nav-link"><%- HtmlUtils.interpolateHtml(
```

Comment:
I think this should use ``StringUtils.interpolate`` since it isn't producing HTML. It is a red flag for me to use ``<%-`` on the result of ``HtmlUtils.interpolateHtml`` because that will end up double-escaping.